docs/getting-started/inferno-cli.md

```diff
@@ -25,10 +24,37 @@ The commands available include:
 | `inferno console` | Starts an interactive console session with Inferno. More information can be found on the [Debugging Page](debugging.html#interactive-console). |
 | `inferno help [COMMAND]` | Describes the available commands, or one specific command if specified |
 | `inferno migrate` | Runs database migrations |
+| `inferno new TEST_KIT_NAME` | Create a new test kit. Run `inferno new --help` for additional options. |
 | `inferno services start` | Starts the background services (nginx, Redis, etc.) for Inferno |
 | `inferno services stop` | Stops the background services for Inferno |
 | `inferno start` | Starts Inferno |
-| `inferno suite SUBCOMMAND ...ARGS` | Performs suite-based operations. The available subcommands are `describe`, `help`, and `input_template`.|
+| `inferno suite SUBCOMMAND [...ARGS]` | Performs suite-based operations. The available subcommands are `describe`, `help`, and `input_template`.|
 | `inferno suite help SUBCOMMAND` | View details on the suite subcommands. |
 | `inferno suites` | Lists available test suites |
 {: .grid.command-table}
+
+## Creating a new Test Kit
+
```

Comment:
I would add a copy-paste code block somewhere here too:
```
inferno new MyTestKit --author "My Name" --author "My Friend" --implementation-guide hl7.fhir.us.core@6.1.0
```

Reply:
added the code block, but will add copy-paste button separately